hail_scripts/shared/elasticsearch_client_v7.py

```diff
@@ -130,3 +130,22 @@ def _update_settings(self, index_name, body):
     def get_index_meta(self, index_name):
         mappings = self.es.indices.get_mapping(index=index_name)
         return mappings.get(index_name, {}).get('mappings', {}).get('_meta', {})
+
+    def wait_for_shard_transfer(self, index_name, num_attempts=1000):
+        """
+        Wait for shards to move off of the loading nodes before connecting to seqr 
+        """
+        # RGP is too large and will not transfer until old index is deleted
+        if "r0384_rare_genomes_project_gen" not in index_name:
```

Comment:
I understand wanting to take a manual step out of the airflow config, but I am worried that this is really fragile. what if we instead checked if it was a WGS project with > 1000 samples? That would basically only affect RGP, but would be more resilient and would probably help if we ever did get another huge WGS project

Reply:
Yeah that is a good point. Do you think 1000 is the right cut off? I'm wondering if we should go off total index size instead? Though I'm not sure if that can be retrieved.

Reply:
I think you are right, total index size would be better than number of samples. Maybe use num shards as a proxy for that, since we have that computed in the pipeline and its based off the number of total variants?

Reply:
Great idea!